### DISCUSSION TITLE
Remove unauthorized CP Widgets

### DISCUSSION THREAD

**Discussion by ARCHES-Academy** [statamic/cms#4501] [Q&A] [answered]
Is there an option to hide the widgets that are not relevant (or authorized) to certain users?

I have a group of users who are not authorized to view any collections or forms. The CP correctly determines that the collection widgets should be hidden, but it still displays the form widgets. This user group should not have access to any forms, nor view any responses (which is why if a user clicks on the widget it says that they are unauthorized).

Is this a bug, feature request, or misconfigured widget?

**Comment by duncanmcclean** [comment, marked as answer]
I would consider it a bug.

**Reply by ARCHES-Academy** [reply]
Sounds good. I submitted an issue for it then. Thanks!

https://github.com/statamic/cms/issues/4506
